frontend/src/components/maps/MineMap.js

```diff
@@ -120,7 +138,25 @@ class MineMap extends Component {
           onLoad={this.handleLoadMap}
         >
           <MinePin />
-        </WebMap>
+        </Map>
+      );
+    }
+    if (this.state.mapFailedToLoad) {
+      return (
+        // Fallback to default map if any of the layers fail to load
+        <Map
+          style={{ width: "100vw", height: "100vh" }}
+          mapProperties={{ basemap: "topo" }}
+          viewProperties={{
+            center: [this.props.long, this.props.lat],
```

Comment:
If lat/long aren't required props then they need default props values. If this component breaks when they're `null` then they should be required.

Commenting here because I need code changes in order to comment on a line.